packages/grafana-data/src/types/data.ts

```diff
@@ -169,7 +169,7 @@ export enum NullValueMode {
  */
 export interface DataConfigSource {
   configRev?: number;
-  dataSupport?: PanelPluginDataSupport;
+  getDataSupport: () => PanelPluginDataSupport;
```

Comment:
Removed this as a property, now PanelModel can just implement this function and return plugin.dataSupport , this change was due to dataSupport being undefined when going into panel edit, and in some or scenarios as well (I think)